### ISSUE TITLE
Accessibility Improvements – Dynamic Alt Text

### ISSUE THREAD

**Issue by sheepysheepy** [img-mapper/react-img-mapper#83]
Map is getting flagged for accessibility issues – namely around labelling for the map and individual area elements. 

https://www.w3.org/WAI/tutorials/images/imagemap/

> For image maps, text alternatives are needed on both the <img> element itself (to convey the informative context) and on each of the <area> elements (to convey the link destination or the action that will be initiated if the link is followed).

Currently, the img element and all area elements are hardcoded with the alt text "map". I'd like to be able to pass in more descriptive alt text for each.

Is it possible to add this functionality?

**Comment by NishargShah** [comment]
Not right now, but will be in the new version

**Comment by NishargShah** [comment]
Resolved in V2.0.0-alpha.8 release, please check `imgProps/areaProps`

**Comment by sheepysheepy** [comment]
Appreciate the response! That works great for setting a custom img alt, but I'm still unable to assign alt text to each area - ideally this would be set per area in the same way that I can set coords or ids. It looks like I can only pass a single alt text string via areaProps that is used across all areas - is there another approach?
